lib/git.rb

```diff
@@ -42,25 +42,9 @@
 # Author::    Scott Chacon (mailto:schacon@gmail.com)
 # License::   MIT License
 module Git
-  
-  #g.config('user.name', 'Scott Chacon') # sets value
```

Comment:
So this is a bit out of scope, but this method is confusing as hell. I don't see anywhere that mixes this in, it's not tested, and it's duplicative of code in `Base`.